cmd/ctr/commands/utils.go

```diff
@@ -25,3 +31,59 @@ func IntToInt32Array(in []int) []int32 {
 	}
 	return ret
 }
+
+// TraverseMap walks the depth of a map and returns the value of the nested keys provided.
+func TraverseMap(m map[string]interface{}, keys ...string) (value interface{}, err error) {
```

Comment:
I don’t think we want this. Why not just dump json and use jq?

Reply:
@AkihiroSuda Thanks for the input.
'jq' is something which is not installed by default on linux machines. For an individual to simply check certain configurations, yes jq helps a lot. But this will not scale for an enterprise, where they have to install a third party tool on every machine, just to read a particular configuration. 
The same argument could be used on docker, on why docker has a 'format' flag.

To give some more context, I come from a cyber security background, and one of the things we do is identify best security practices for configurations (like CIS benchmarks). Evaluating these configurations should be a trivial task, where someone runs a single command and gets to know what their configuration is, for a particular field. Installing a new tool like jq to do that job would be a pre-requisite which most people would not prefer to evaluate best practices. Hence the introduction of adding such in-built functionalities within containerd. Let me know if that makes sense?

Reply:
If jq is not available the user can use python’s json parser or something else similar

Reply:
I’m not against implementing --format flag but it should be implemented using Go’s `text/template` so that you do not need to implement this traverser function